chapter_3_practic/task11.cpp

```diff
@@ -0,0 +1,84 @@
+﻿// Create a three - function calculator for old - style English currency, where money amounts
+// are specified in pounds, shillings, and pence. (See Exercises 10 and 12 in Chapter 2.)
+// The calculator should allow the user to add or subtract two money amounts, or to multiply a money amount by a floating - point number. 
+// (It doesn’t make sense to multiply two
+// money amounts; there is no such thing as square money.We’ll ignore division.Use the
+// general style of the ordinary four - function calculator in Exercise 4 in this chapter.)
+
+#include <iostream>
+
+int main()
+{
+	char repeat = 'y';
+	char sign = ' ';
+
+	while (repeat == 'y')
+	{
+		int  pounds1 = 0, shillings1 = 0, pence1 = 0;
+		int  pounds2 = 0, shillings2 = 0, pence2 = 0;
+		int  pounds = 0, shillings = 0, pence = 0; // for results
+		float multiplay = 0.0f;
+		char ch = ' ';
+
+		std::cout << "Enter first amount: (5.10.6)"; // Запрос ввода пользователем первого значения
+		std::cin >> pounds1 >> ch >> shillings1 >> ch >> pence1;
+
+		std::cout << "Enter operation '+', '-' or '*'";// Выбор операции: сложение, вычитание, умножение и запрос на ввод значений
+		std::cin >> sign;
+		if (sign != '*')
```

Comment:
if you restructure the code like
if ('*' == sign)
{
// do for multiplier
}
it's easier to read in comparison to when you have != operator
Also, prefer the style 
if (/*constant*/ value == changeable)
in this case means if ('*' == value)
such style prevents the case if (value = '*') which you could make by mitake

Reply:
https://github.com/Dmitriy-Romanchuk/master/pull/3/commits/90cb98bbed2eadee4be5c4c4c8b743da3a1e4554